{{cookiecutter.python_name}}/{{cookiecutter.python_name}}/__init__.py

```diff
@@ -38,7 +36,8 @@ def _load_jupyter_server_extension(server_app):
         JupyterLab application instance
     """
     setup_handlers(server_app.web_app)
-    server_app.log.info("Registered HelloWorld extension at URL path /{{ cookiecutter.python_name | replace('_', '-') }}")
+    server_app.log.info("Registered %s server extension" % data["name"])
```

Comment:
Let's use modern Python

```suggestion
    server_app.log.info("Registered {name} server extension".format(**data))
```